x-pack/legacy/plugins/siem/public/components/open_timeline/index.test.tsx

```diff
@@ -85,26 +77,15 @@ describe('StatefulOpenTimeline', () => {
           </TestProviderWithoutDragAndDrop>
         </ThemeProvider>
       );
-
-      await wait();
-      wrapper.update();
-
       wrapper
         .find('[data-test-subj="search-bar"] input')
         .simulate('keyup', { keyCode: 13, target: { value: '   abcd   ' } });
-
-      wrapper.update();
-
-      expect(getStateChildComponent(wrapper).state).toEqual({
```

Comment:
I liked it when we were making sure than our store get updated

Reply:
this was checking react state, not the redux store, which is no longer being used